analyticsdataserver/settings/base.py

```diff
@@ -376,3 +396,5 @@
 }
 
 ########## EDX ENTERPRISE DATA CONFIGURATION
+API_AUTH_TOKEN = 'put-your-api-token-here'
+CSRF_COOKIE_SECURE = False
```

Comment:
You need to include all these values as well:
https://github.com/edx/configuration/blob/master/playbooks/roles/edx_django_service/defaults/main.yml#L182-L222

as they get merged into the edx_django_service_config here:
https://github.com/edx/configuration/blob/master/playbooks/roles/edx_django_service/defaults/main.yml#L226